setup.py

```diff
@@ -48,6 +48,42 @@
 It should work without additional code in any environment with
 Python >= 2.4, ctypes and an pre-built usb backend library
 (currently, libusb 0.1.x, libusb 1.x, and OpenUSB).
-"""
+""",
+    classifiers=[
+        'Development Status :: 4 - Beta',
```

Comment:
In the PyPi trove classifier scheme, the next status up (5) is "Production/Stable". I can see that the mapping between the semver.org scheme and the trove scheme might not be exactly 1-to-1, but there's probably a mechanical way to do it. I would think that anything _up to_ a 1.x release in the semver scheme, including a candidate, would still be fit into 'beta' in the trove scheme (unless the semver tag is -alpha* or something).  I'm not sure how that would apply to, say, 1.3.0-rc1, but that's a ways off....

I've looked around at techniques for reducing the redundancy - many seem to recommend that setup.py should avoid actually importing the module, but should simply parse either `__init.py__` or a dedicated version file for the version information only. I wasn't sure what you'd want to do there.